src/components/Login.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+import Button from './Button';
+import ErrorCard from './ErrorCard';
+import Input from './Input';
+
+export default function Login() {
+  const [error, setError] = useState(null);
+  const [credentials, setCredentials] = useState({
+    email: '',
+    password: '',
+  });
+  const handleSubmit = async (e:any) => {
+    e.preventDefault();
+
+    console.log(credentials);
```

Comment:
Please do not leave console logs. And remember not to log credentials in any scenario.

Reply:
Okey!